src/actions/vstudio/vs2010_vcxproj.lua

```diff
@@ -1247,6 +1247,8 @@
 			if value then
 				m.element("DebugInformationFormat", nil, value)
 			end
+		elseif cfg.symbols == p.OFF then
+			m.element("debugInformationFormat", nil, "None")
```

Comment:
`DebugInformationFormat` not `debugInformationFormat`.

Also, I believe `None` was introduced in VS2015 so this will need to check for that.